src/components/network-modification-tree-pane.js

```diff
@@ -87,4 +93,8 @@ export default NetworkModificationTreePane;
 
 NetworkModificationTreePane.propTypes = {
     studyUuid: PropTypes.string.isRequired,
+    drawerNodeEditorOpen: PropTypes.bool.isRequired,
```

Comment:
Maybe remove these props and store the drawerNodeEditorOpen prop in redux store instead.

Reply:
Done